gateway/api/context.py

```diff
@@ -0,0 +1,53 @@
+"""User context management using ContextVars."""
+
+from contextvars import ContextVar
+from contextlib import contextmanager
+from typing import Optional
+from django.contrib.auth.models import User  # pylint: disable=imported-auth-user
```

Comment:
This pylint is correct, the recommended way from Django to import the `User` model is through the method `get_user_model`:
- Django doc [link](https://docs.djangoproject.com/en/5.2/topics/auth/customizing/#django.contrib.auth.get_user_model)
- [Example](https://github.com/Qiskit/qiskit-serverless/blob/main/gateway/api/repositories/users.py#L15) from our repository